From 71549d035f18c5e53e25f2b66f8e795dff61ab37 Mon Sep 17 00:00:00 2001
From: Marcelo Leitner <mleitner@redhat.com>
Date: Tue, 21 Oct 2014 16:19:14 -0400
Subject: [net] netfilter: nft_log: fix coccinelle warnings

Message-id: <20141021161914.22888.74432.stgit@localhost.localdomain>
Patchwork-id: 97973
O-Subject: [RHEL7 net PATCH 077/124] netfilter: nft_log: fix coccinelle warnings
Bugzilla: 1148041 1155088
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

Backport of upstream commit: 5cbfda2043814f4989efad12be02086a2e4c59fd

 commit 5cbfda2043814f4989efad12be02086a2e4c59fd
 Author: Fengguang Wu <fengguang.wu@intel.com>
 Date:   Sun Jun 29 13:55:08 2014 +0200

    netfilter: nft_log: fix coccinelle warnings

    net/netfilter/nft_log.c:79:44-45: Unneeded semicolon

     Removes unneeded semicolon.

    Generated by: scripts/coccinelle/misc/semicolon.cocci

    CC: Pablo Neira Ayuso <pablo@netfilter.org>
    Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>

Signed-off-by: Marcelo Ricardo Leitner <mleitner@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/netfilter/nft_log.c b/net/netfilter/nft_log.c
index 5b1a4f5..bde05f2 100644
--- a/net/netfilter/nft_log.c
+++ b/net/netfilter/nft_log.c
@@ -76,7 +76,7 @@ static int nft_log_init(const struct nft_ctx *ctx,
  case NF_LOG_TYPE_LOG:
   if (tb[NFTA_LOG_LEVEL] != NULL) {
    li->u.log.level =
-    ntohl(nla_get_be32(tb[NFTA_LOG_LEVEL]));;
+    ntohl(nla_get_be32(tb[NFTA_LOG_LEVEL]));
   } else {
    li->u.log.level = 4;
   }
-- 
1.7.1